guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

```diff
@@ -46,13 +48,12 @@ endif::[]
 
 You must do this when {ProjectServer} or {SmartProxyServer} has different internal and external DNS host names and there is no site-to-site VPN connection between the locations where you deploy {ProjectServer} and {SmartProxyServer}.
 
-ifndef::foreman-deb[]
+ifdef::katello,orcharhino,satellite[]
 .On demand content sources
 You can use the *On demand* download policy to reduce the storage footprint of the Red Hat Enterprise Linux server that runs {Project}.
 When you set the download policy to *On Demand*, content syncs to {ProjectServer} or {SmartProxyServer} when a content host requests it.
 
 For more information, see {ContentManagementDocURL}Importing_Content_content-management[Importing Content] in the _Content Management Guide_.
-
 endif::[]
 
 == Use Cases that Do Not Work
```

Comment:
@lzap Wouldn't these use cases actually work if you deploy a Smart Proxy in your network? In Scenario 2 in this guide they do so which means I don't understand why this is listed as Do Not Work.

Reply:
No idea I am trying to stay away from compute resources.

Reply:
But this whole comment is weird indeed, it is pretty obvious that infrastructure in IaaS is managed by the cloud.

Reply:
> No idea I am trying to stay away from compute resources.

This whole guide is about running Foreman on AWS, not managing AWS from within Foreman.